apps/i18n/common/en_us.json

```diff
@@ -513,7 +513,7 @@
   "deleteProjectConfirm": "Do you really want to delete your project? You cannot undo this action.",
   "deleteSection": "Delete Section",
   "deleteSectionConfirm": "Are you sure you want to delete this section? You will not be able to undo this.",
-  "deleteSectionHideSuggestion": "If you simply want to remove this section from your list of sections, consider using the 'Hide section' option instead.",
+  "deleteSectionArchiveSuggestion": "If you simply want to remove this section from your list of sections, consider using the 'Archive section' option instead.",
```

Comment:
Just double-checking that this is intentional:

Adding a new string for this and removing the old one (as opposed to updating the old string) will result in any existing translations for this string being discarded, and the string reverting to English everywhere until it can be retranslated. Is this the desired behavior?

Reply:
My thought was that since it's not a super visible string, that would be okay. But if that's not the preferred way to handle this, I can keep the old one in use until we get translations for the new one - how do long do we usually wait?

Reply:
I also thought that existing translations would not get updated if the key didn't change. Is that incorrect?

Reply:
It definitely can be okay, particularly if this isn't a super visible string. If you're not sure, checking with the international team is probably the best way to find out; @castro-jorge can help you with both that and figuring out how long it will take to translate.

You're correct that existing translations won't get automatically updated if the key doesn't change, but they will get flagged as needing re-translation, and will get updated by our translators eventually.

Reply:
Yeah, it seems like the only who would view that string are teachers anyway, so I am comfortable making the change. I'll be submitting a batch of strings by the end of the week and I'll add this one to the list. Translation of new strings typically takes 2 weeks.

Reply:
Thanks @Hamms and @castro-jorge! I'll go ahead and merge it. It's only visible to teachers in the very specific scenario where they have a section with no students and attempt to delete the section.